static/src/js/components/AdminPreferencesMetrics/__tests__/AdminPreferencesMetrics.test.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react'
+import ReactDOM from 'react-dom'
+import { render, screen } from '@testing-library/react'
+
+import '@testing-library/jest-dom'
+
+import { BrowserRouter } from 'react-router-dom'
+
+import { AdminPreferencesMetrics } from '../AdminPreferencesMetrics'
+
+const setup = () => {
+  const metricsPreferences = {
+    isLoaded: true,
+    isLoading: false,
+    preferences: {
+      panelState: {},
+      granuleSort: {},
+      granuleListView: {},
+      collectionSort: {},
+      collectionListView: {},
+      zoom: {},
+      latitude: {},
+      longitude: {},
+      projection: {},
+      overlayLayers: {},
+      baseLayer: {}
+    }
+  }
+  const onFetchAdminMetricsPreferences = jest.fn()
+
+  const props = {
+    onFetchAdminMetricsPreferences,
+    metricsPreferences
+  }
+
+  // https://testing-library.com/docs/example-react-router/
+  render(<AdminPreferencesMetrics {...props} />, { wrapper: BrowserRouter })
```

Comment:
React Router recommends using MemoryRouter in tests, https://reactrouter.com/en/v6.3.0/api#memoryrouter